bin/aws/ssm_nomad_agent.sh

```diff
@@ -1,24 +1,16 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
-
-PORT_TO_FORWARD="${1:-1234}"
+# This script sets up port forwarding between an arbitrary nomad agent server and your local computer.
+# This is a temporary workaround until we set up a VPN
+# This will allow you to view the UI of a container on the server and/or interact with the container via the cli
+#
+# Input (optional): Port to forward to. This will be both the port on the server to forward and the local port that is forwarded to. Defaults to port 1234
 
-SSM_PARAMETERS=$(
-    cat << EOF
-{
-  "portNumber": ["${PORT_TO_FORWARD}"],
-  "localPortNumber": ["${PORT_TO_FORWARD}"]
-}
-EOF
-)
+set -euo pipefail
 
-AGENTS=$(aws ec2 describe-instances --filter Name=tag:Name,Values="Nomad Agent")
-NOMAD_AGENT_INSTANCE_ID=$(echo "${AGENTS}" | jq -r .Reservations[0].Instances[0].InstanceId)
+# shellcheck source=bin/aws/lib/ssm_tools.sh
+source lib/ssm_tools.sh
 
-echo "--- Instance: ${NOMAD_AGENT_INSTANCE_ID}"
+PORT_TO_FORWARD="${1:-1234}"
```

Comment:
Is this more to access the Grapl UI rather than the Nomad agent _per se_?

Reply:
Yes. I've added an array to make this clearer in the user-facing messaging